troposphere/static/js/components/modals/instance/image/components/Tags.react.js

```diff
@@ -121,10 +131,10 @@ export default React.createClass({
               For your convenience, we've automatically added the tags that were already on the instance.
```

Comment:
Do we want to put escaping out the `'` in this text? I believe that the red marking is just Git/GitHub's text-tools being angry. But I think we could do a `{" ... "}` here.